vendor/bat-native-ads/src/bat/ads/internal/frequency_capping/exclusion_rules/total_max_frequency_cap_unittest.cc

```diff
@@ -0,0 +1,157 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <memory>
+
+#include "testing/gtest/include/gtest/gtest.h"
+
+#include "base/time/time.h"
+
+#include "bat/ads/internal/frequency_capping/exclusion_rule.h"
+#include "bat/ads/internal/frequency_capping/frequency_capping.h"
+#include "bat/ads/internal/frequency_capping/exclusion_rules/total_max_frequency_cap.h"
+
+#include "bat/ads/internal/client_mock.h"
+#include "bat/ads/internal/ads_client_mock.h"
+#include "bat/ads/internal/ads_impl.h"
+#include "bat/ads/ad_info.h"
+
+// npm run test -- brave_unit_tests --filter=Ads*
+
+using std::placeholders::_1;
+using ::testing::_;
+using ::testing::Invoke;
+
+namespace ads {
+
+
+const char test_creative_set_ids[][40] = {
+    "654f10df-fbc4-4a92-8d43-2edf73734a60",
+    "465f10df-fbc4-4a92-8d43-4edf73734a60"
+};
+
+class BraveAdsTotalMaxFrequencyCapTest : public ::testing::Test {
+ protected:
+  BraveAdsTotalMaxFrequencyCapTest()
+  : mock_ads_client_(std::make_unique<MockAdsClient>()),
```

Comment:
nit: indent a further 2 spaces for this and the following line

Reply:
Done